### PR TITLE
Relax sqlx dependency

### PR DESCRIPTION
**Pull request by LecrisUT**
Cherry-picking from the last PR so that things are easier when the next release is cut

## Checks
- [x] I am happy for maintainers to push small adjustments to this PR, to speed up the review cycle
- [x] I have checked that there are no existing pull requests for the same thing

### CODE REVIEW

File: ui/backend/Cargo.toml [hunk 1]
```diff
@@ -27,7 +27,7 @@ syntect = "5.2.0"
 tauri-plugin-http = "2.0.0-beta"
 
 [dependencies.sqlx]
-version = "=0.7.3"
+version = "0.7.3"
```
Comment (ellie) [inline ui/backend/Cargo.toml:30]:
```suggestion
version = "0.7.4"
```

Comment (LecrisUT) [reply ui/backend/Cargo.toml:30]: What do you think of `0.7` instead and relaxing some other ones as well?

Comment (ellie) [reply ui/backend/Cargo.toml:30]: That would be fine

Comment (LecrisUT) [reply ui/backend/Cargo.toml]: Ok, just have 2 of them now if you're ok with them

**Review by ellie — APPROVED** [review]
🚀

**Pull request merged**